README.md

```diff
@@ -32,7 +32,7 @@ Before installing **powsybl-open-rao**, you need to install OR-Tools, a software
 
 Please refer to [OR-tools website](https://developers.google.com/optimization/install/download) for installation instructions.
 
-Open RAO also needs a load flow implementation and a sensitivity analysis implementation at runtime, following the interfaces of **powsybl-core** which documentation is available in [PowSyBl website](https://www.powsybl.org/pages/documentation/). Note that for obvious reasons, included performances, reliability and transparency, Open RAO uses [Powsybl Open Load Flow] by default, but you can prefer you own implementation.
+Open RAO also needs a load flow implementation and a sensitivity analysis implementation at runtime, following the interfaces of **powsybl-core** which documentation is available in [PowSyBl website](https://www.powsybl.org/pages/documentation/). Note that for obvious reasons, included performances, reliability and transparency, Open RAO uses [Powsybl Open Load Flow](https://github.com/powsybl/powsybl-open-loadflow) by default, but you can prefer you own implementation.
```

Comment:
we only use open load flow in open RAO to be able to run some unit tests, maybe it is worth mentioning